ichnaea/log.py

```diff
@@ -25,29 +26,43 @@ def configure_logging():
     logging_level = settings("logging_level")
 
     if local_dev_env:
-        handlers = ["console"]
+        handlers = ["dev"]
+        # Prepare structlog logs for local dev ProcessorFormatter
+        structlog_fmt_prep = structlog.stdlib.ProcessorFormatter.wrap_for_formatter
     else:
         handlers = ["mozlog"]
-
+        # Prepare structlog logs for JsonLogFormatter
+        structlog_fmt_prep = structlog.stdlib.render_to_log_kwargs
+
+    # Processors used for logs generated by structlog and stdlib's logging
+    structlog_shared_processors = [
+        structlog.stdlib.add_logger_name,
+        structlog.stdlib.add_log_level,
+        structlog.processors.TimeStamper(fmt="iso"),
```

Comment:
This `iso` format is different from the `logging` `asctime` format, but has similar data for about the same amount of effort:

```
structlog: 2020-04-29T15:42:56.725306Z
  logging: 2020-04-29 15:42:56,725
```

This ``structlog_shared_processors`` is used to ensure that both ``structlog`` and stdlib ``logging`` log items have the same elements in the dev environment. ``JsonLogFormatter`` does similar work for non-dev deployments.